avm/ptn/alz/empty/modules/policy-set-definitions/README.md

```diff
@@ -0,0 +1,55 @@
+# avm/ptn/alz/empty/policy-set-definitions `[Alz/EmptyModulesPolicySetDefinitions]`
```

Comment:
Hey @jtracey93,
did you generate these manually? Usually the files in the `/modules` folder are only considered 'helper templates' that don't have their own readme. Just curious. The `Set-AVMModule` script should have ignored them and maybe didn't because you build them up like child modules with a main.bicep and all.

What is usually expected in the `/modules`  folder would simply be a structure like
- alz/empty
  - main.bicep
  - main.json
  - readme.md
  - /modules
    - policyDefinition.bicep
    - policySetDefinitions.bicep
    - wait.bicep   

So no extra folders and `main.bicep` files. I'm not sure what the spec says as I wasn't able to find it anymore, but I guess you approach is fine too. Just wanted to point it out.

Reply:
Ah okay ill adjust the file/folder structure. But for clarity, yes the `set-avmmodule` script did these, not me 👍